src/main/java/com/cairn/waypoint/dashboard/endpoints/steptemplate/dto/HomeworkTemplateDetailsDto.java

```diff
@@ -9,4 +14,13 @@ public class HomeworkTemplateDetailsDto {
 
   private Long id;
   private String name;
+
+}
+
+@Mapper
+interface StepTemplateLinkedHomeworkTemplateMapper {
```

Comment:
I see what you've done here. You've utilized a mapper here inside the class. This could be a maintenance hotspot, especially on larger teams/as the team grows. Could we instead move the code here into a separate `*.java` file in the adjacent `.mapper` package?